Documentation/ceph-csi-drivers.md

```diff
@@ -124,5 +124,77 @@ when pod get spawned and destroyed at time of pod delete.
 Refer to [ephemeral-doc](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes) for more info.
 Also, See the example manifests for an [RBD ephemeral volume](https://github.com/rook/rook/tree/{{ branchName }}/deploy/examples/csi/rbd/pod-ephemeral.yaml) and a [CephFS ephemeral volume](https://github.com/rook/rook/tree/{{ branchName }}/deploy/examples/csi/cephfs/pod-ephemeral.yaml).
 
+## CSI-Addons Controller
+
+The CSI-Addons Controller handles the requests from users to initiate an operation. Users create a CR that the controller inspects, and forwards a request to one or more CSI-Addons side-cars for execution.
+By listing the CSIAddonsNode CRs, the CSI-Addons Controller knows how to connect to the side-cars. By checking the supported capabilities of the side-cars, it can decide where to execute operations that the user requested.
+
+### Installation by operator
+
+A CSI-Addons bundle can be used to install the CSI-Addons Controller with the following steps:
```

Comment:
Installing the bundle is a black box to the rook users. How about a link to the csiaddons that tells more details about what will be created in the namespace?